### PR TITLE
Matt nav fix

### PR DESCRIPTION
**Pull request by mnolan34**
The Nav bar is fixed, allowing for multiple searches without breaking the application

### CODE REVIEW

File: src/tests/tags-service.test.js [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    // Set up tests
+    beforeAll(async () => {
+        let promises = [];  // Initialize an empty list of promises
+
+        /*
+        Add promises to remove users with test username and
+        remove tuits with test texts to list of promises
+        */
+        promises.push(deleteUsersByUsername(sonic.username));
+        promises.push(deleteTuitByTuitText(mockedTuit1.tuit));
```
Comment (mcartervt) [inline src/tests/tags-service.test.js]: In the latest version, the deleteTuitByTuitText() calls have a await in front of them.  Same in afterAll.

File: src/tests/tags-service.test.js [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        testTuit3 = await findTuitById(tid3);
+
+        let promises = [];  // Initialize an empty list of promises
+
+        /*
+        Add promises to remove users with test username and
+        remove tuits with test texts to list of promises
+        */
+        promises.push(deleteUsersByUsername(sonic.username));
+        promises.push(deleteTuitByTuitText(mockedTuit1.tuit));
```
Comment (mcartervt) [inline src/tests/tags-service.test.js]: Add await in front of these to match beforeAll.

**Review by mcartervt — CHANGES_REQUESTED** [review]
Will approve if you use the updated tests.  tags-service.test.js

Also, I don't know that you have tuit2tags-service.test.js either?

**Comment by kachalmers** [comment]
> Will approve if you use the updated tests. tags-service.test.js
> 
> Also, I don't know that you have tuit2tags-service.test.js either?

Agreed: In case there is overlap with the work, I would recommend a merge of the main branch into this branch to see if there are any conflicts before approving these updates/merging this branch into the main branch.

**Comment by mnolan34** [comment]
Agreed, want to keep the react service tests as done by mark.

**Review by mcartervt — APPROVED** [review]

**Review by mcartervt — APPROVED** [review]
Re-tested.  Can see my 2 test files match current versions.  Still can do multiple searches.

Approved!

**Review by kachalmers — APPROVED** [review]

**Pull request merged**